### PR TITLE
feat: initial rebranding and added pagination for tabs component

### PR DESCRIPTION
**Pull request by r-mulder**
This PR will update the tabs component to the new brand update.

Also included:
- Pagination for the tabs list. 
- focus on active item when rendered.
- Support scroll and pagination on all devices.

### CODE REVIEW

**Review by eileenmguo — COMMENTED** [review]
looks like the border doesn't go all the way across the tab container
image

Also, I don't see an example of pagination in storybook.

File: packages/libs/react-ui/src/components/Tabs/Tabs.css.ts [hunk 1]
```diff
@@ -85,13 +59,89 @@ export const selectorLine = style([
...
+          color: token('color.text.base.@active'),
+        },
+        '&[data-hovered="true"]': {
+          color: token('color.text.base.@hover'),
+        },
+      },
+    },
+  ],
+  variants: {
+    inverse: {
```
Comment (ferreroltd) [inline packages/libs/react-ui/src/components/Tabs/Tabs.css.ts:100]: I wonder if this was the dark/light mode things, probably not but still curious

Comment (r-mulder) [reply packages/libs/react-ui/src/components/Tabs/Tabs.css.ts:100]: No this is to support a nested structure, A UX pattern is to have a lighter background color for the active tab so the inner tabs component can have the background color of the outer active tab.

**Comment by eileenmguo** [comment]
image
 The tabs look a bit broken here. Is this the intended design? @isaozler

**Review by isaozler — APPROVED** [review]

**Review by eileenmguo — COMMENTED** [review]
Code looks good! 

The storybook looks a bit broken because the close button doesn't do anything. Could you add some state management so that the close button works in the example?

**Review by sstraatemans — APPROVED** [review]

**Pull request merged**